pkgs/top-level/all-packages.nix

```diff
@@ -23520,12 +23520,12 @@ with pkgs;
 
   clamsmtp = callPackage ../servers/mail/clamsmtp { };
 
-  clickhouse = callPackage ../servers/clickhouse {
-    # upstream requires llvm12 as of v22.3.2.2
+  inherit (callPackage ../servers/clickhouse {
+    # upstream requires llvm13 as of v22.8.5.29
     inherit (llvmPackages_13) clang-unwrapped lld llvm;
     llvm-bintools = llvmPackages_13.bintools;
     stdenv = llvmPackages_13.stdenv;
-  };
+  }) clickhouse clickhouse-lts;
```

Comment:
The pattern I've seen elsewhere in this file is to define a core `.nix` file with input parameters like `version_suffix` (which should probably be renamed to `version-suffix`), and then invoke that twice to get two separate packages, like this:

```nix
clickhouse = callPackage ../servers/clickhouse/core.nix {
  version-suffix = "stable";
  ...
};

clickhouse-lts = callPackage ../servers/clickhouse/core.nix {
  version-suffix = "lts";
  ...
};
```

Reply:
I'm not sure, then `hash` should be put in `all-packages.nix`. But I see no one puts hash there hmmm

Reply:
I took this example from: https://github.com/NixOS/nixpkgs/blob/master/pkgs/servers/hbase/default.nix

Reply:
We can do that. Repeating the path twice is not that nice.